pkg/destroy/openstack/openstack_deprovision.go

```diff
@@ -0,0 +1,187 @@
+package openstack
+
+import (
+	"os"
+	"time"
+
+	"github.com/openshift/installer/pkg/destroy"
+	"github.com/openshift/installer/pkg/types"
+
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/openstack/compute/v2/servers"
+	"github.com/gophercloud/utils/openstack/clientconfig"
+	"github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+// OpenStackFilter holds the key/value pairs for the tags we will be matching against
```

Comment:
nit: This is missing trailing period.

Some of your other godocs comments are as well.  Obviously not the end of the world, but should be easy to fix ;).

And we probably have a number of godocs in master with the same issue; PRs welcome :).

Reply:
Ah thanks, I'll go through and check - is there a way to build the godocs locally so I can check the format before pushing?